lib/utils.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable max-len,no-shadow,no-param-reassign,no-underscore-dangle,no-use-before-define,consistent-return */
+/* eslint-disable max-len,no-shadow,no-param-reassign,no-underscore-dangle,no-use-before-define,consistent-return,no-trailing-spaces,space-infix-ops */
```

Comment:
My logic was since the document already contained 6 rules which were disabled, it would be appropriate to add 3 more, but of course that might have not been valid